pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -584,16 +585,11 @@ func (p *criStatsProvider) addSwapStats(
 
 func (p *criStatsProvider) addProcessStats(
 	ps *statsapi.PodStats,
-	podUID types.UID,
-	allInfos map[string]cadvisorapiv2.ContainerInfo,
-	cs *statsapi.ContainerStats,
+	container *cadvisorapiv2.ContainerInfo,
 ) {
-	// try get process stats from cadvisor only.
-	info := getCadvisorPodInfoFromPodUID(podUID, allInfos)
-	if info != nil {
-		ps.ProcessStats = cadvisorInfoToProcessStats(info)
-		return
-	}
+	processStats := cadvisorInfoToProcessStats(container)
```

Comment:
can you add this to cadvisor_stats_provider?

Reply:
Good idea. Updated.

Reply:
With https://github.com/kubernetes/kubernetes/pull/123369/commits/baa7e4b44d3905a374f5b0409ce915d85836c0f0, the crio e2e still failed.

Reply:
Do we have to fix the that in this PR as well? This at least will fix the containerd CI.

Reply:
Yea, feel free to drop the fix in cadvisor stats provider. I’ll open it up and investigate it on my end.